### ISSUE TITLE
[ERROR]: gitstatus failed to initialize. Re-opening. 

### ISSUE THREAD

**Issue by edgarnegronrabell** [romkatv/gitstatus#88]
First of all, I apologize for not responding earlier, I've had quite a busy week and was having a difficult time finding the document after I passed the commands you told me.

As for the problem, it started sometime after updating to OS Catalina, I think it happened at v.15.2. Most of the times I open the terminal, I usually open it through VS Code, but the problem occurs when I open the normal terminal (I use iTerm), I get the and error message and a command that gives me extra information.

**Comment by romkatv** [comment]
Do you get this error every time you start zsh from iTerm or only sometimes?

When you get the error, does it appear instantly after starting zsh or after 5 seconds?

Your config is attempting to use two themes at one. You need to remove this line:

```zsh
source ~/powerlevel9k/powerlevel9k.zsh-theme
```

Does this help? If not, you should still remove this line.

**Comment by edgarnegronrabell** [comment]
Hi, I will try this out. I've been having problems with my computer ever since upgrading to OS Catalina, but this has been happening after the most recent minor update. It used to happen more frequently but I recently deleted xCode, (apparently, I had to different xCodes installed on my machine) and the computer has been running more smoothly overall, so this message hasn't been appearing that much in the past few days. I will remove this line because its probably been doing double the work this whole time. Thanks so much.
